quickstart.md

```diff
@@ -270,12 +270,284 @@ In case the user wants to run the workload on a TDX capable hardware, using QEMU
 
 ### SEV
 
-`kata-qemu-sev`
-Export cert chain
-Start KBS (even for unencrypted)
+#### Install sevctl and Export SEV Certificate Chain
 
-Use the sample image or an unencrypted image
+[sevctl](https://github.com/enarx/sevctl) is the SEV command line utility and is needed to export the SEV certificate chain.
```

Comment:
```suggestion

On Fedora / CentOS / RHEL, there is an `sevctl` package, so you can install the necessary tool with `dnf install -y sevctl`.
```

Reply:
Did this a little differently. Added two sections for Ubuntu vs RHEL path.